src/taipy/config/_config_comparator.py

```diff
@@ -26,21 +28,30 @@ class _ConfigComparator(dict):
     ```python
     {
         "added_items": [
-            ((entity_name_1, entity_id_1, attribute_1), added_object_1),
-            ((entity_name_2, entity_id_2, attribute_2), added_object_2),
+            ((section_name_1, config_id_1, attribute_1), added_object_1),
+            ((section_name_2, config_id_2, attribute_2), added_object_2),
         ],
         "removed_items": [
-            ((entity_name_1, entity_id_1, attribute_1), removed_object_1),
-            ((entity_name_2, entity_id_2, attribute_2), removed_object_2),
+            ((section_name_1, config_id_1, attribute_1), removed_object_1),
+            ((section_name_2, config_id_2, attribute_2), removed_object_2),
         ],
         "modified_items": [
-            ((entity_name_1, entity_id_1, attribute_1), (old_value_1, new_value_1)),
-            ((entity_name_2, entity_id_2, attribute_2), (old_value_2, new_value_2)),
+            ((section_name_1, config_id_1, attribute_1), (old_value_1, new_value_1)),
+            ((section_name_2, config_id_2, attribute_2), (old_value_2, new_value_2)),
         ],
     }
     ```
     """
 
+    _ADDED_ITEMS_KEY = "added_items"
+    _REMOVED_ITEMS_KEY = "removed_items"
+    _MODIFIED_ITEMS_KEY = "modified_items"
+
+    _BLOCKED_SECTION_KEY = "blocked_diff_sections"
```

Comment:
I don't like the naming Block/blocked section. What about something like "unconflicted_sections"
and the method name could be `_unconfilct(cls, Section.name: Union[str, Set[str]])`

Reply:
"unconflict" is not really a verb. `unconflicted_sections` is great, but I'll propose to keep the `_unblock()` method name.

Reply:
`_add_unconflicted_section()` as replace for `_unblock()` method